src/widgets/calendar/CalendarDay.tsx

```diff
@@ -22,12 +24,24 @@ export const CalendarDay = ({ date, medias }: CalendarDayProps) => {
       withinPortal
       radius="lg"
       shadow="sm"
-      transition="pop"
+      transitionProps={{
+        transition: 'pop',
+      }}
       onClose={close}
       opened={opened}
     >
       <Popover.Target>
-        <Box onClick={open}>
+        <Box
+          onClick={open}
+          style={{
```

Comment:
https://mantine.dev/styles/sx/#subscribe-to-theme

Reply:
Hmm is this now outdated or not? GitHub says this is the actual code, but my comment is labeled as "Outdated". Anyhow, if this has been changed, feel free to merge